experiments/ClimaCore/heat-diffusion/run.jl

```diff
@@ -301,8 +301,8 @@ sol_atm, sol_lnd = integ_atm.sol, integ_lnd.sol;
 # `parent(sol_atm.u[<time-index>].atm_F_sfc)[<z-index>,<variable-index>]`
 
 ENV["GKSwstype"] = "nul"
-import Plots
-Plots.GRBackend()
+import Makie
+import CairoMakie
 
 ARTIFACTS_DIR = joinpath("experiments/ClimaCore/output/heat-diffusion_artifacts")
```

Comment:
There are no artifacts on buildkite for the heat diffusion run because this should be `"/heat-diffusion/artifacts"`, but after rebasing on main it should be fine